src/parser/jobs/sch/modules/DoTs.js

```diff
@@ -1,78 +1,39 @@
 import React, {Fragment} from 'react'
```

Comment:
I didn't realise this was both sch and ast for a sec there and got super confused :notlikemeow: